config.yml

```diff
@@ -38,18 +38,5 @@ policies:
 - name: TEDPolicy
   constrain_similarities: true
 - name: FlowPolicy
-- name: EnterpriseSearchPolicy
```

Comment:
You don't need to remove `EnterpriseSearchPolicy` completely. You can just comment out the qdrant part. Also if you remove that part, we should update the README as the qdrant setup is currently part of the default setup.

Reply:
ah, because multi step does not support enterprise search... 💡